### DISCUSSION TITLE
automatic registering blocks from current_theme/blocks folder

### DISCUSSION THREAD

**Discussion by kubamarkiewicz** [wintercms/wn-blocks-plugin#27] [Ideas]
I have an idea, it could be nice to have an alternative way of registering blocks by placing .block files in `current_theme/blocks` folder. Blocks from that folder could be registered automatically. That way it is not necessary to create a plugin in order do define custom blocks. 

What do you think?

**Comment by bennothommo** [comment]
@kubamarkiewicz that should already be happening. The Blocks plugin checks the theme for a `blocks` directory and will include all `.block` files within as available blocks.

**Reply by kubamarkiewicz** [reply]
Yes, it is working! My mistake, I didn't read the documentation carefully.
